rcfile.rc

```diff
@@ -0,0 +1,805 @@
+glyph += $helpful:ring of (fire|ice)
+glyph += $useless:ring of (flight|loudness)
+glyph += $good:ring of (.*evasion|invisibility|magical power|.*protection|stealth|sustain abilities|wizardry)
+glyph += $verygood:ring of (poison resistance|protection from cold|protection from fire|protection from magic|see invisible)
+glyph += $important:ring of (teleport|.*slaying|sustain abilities)
+
+
+##################### Stats colouring (numbers)
+
+status_caption_colour = green
+hp_colour = 100:lightgreen, 90:green, 60:yellow, 30:lightred, 15:red
+stat_colour = 125:white, 50:lightgreen, 14:green, 9:yellow, 6:lightred, 3:red
+mp_colour = 100:lightcyan, 85:cyan, 60:lightblue, 30:blue, 15:red
+
+# Tags: stats, colours, colouring, menu, glyphs, hp, mp, health, mana,
+#####################
+##################### End of colouring
+#####################
+
+
+#####################
+##################### Mechanics
+##################### 
+
+confirm_butcher = never
+easy_eat_chunks = true
+auto_eat_chunks = true
+auto_sacrifice = true
+rest_wait_both = true
+
+travel_delay = -1
+explore_delay = -1
+rest_delay = -1
+
+explore_stop = items,stairs,shops,altars,portals,branches,runed_doors
+travel_avoid_terrain = deep water
+
+easy_unequip = true
+equip_unequip = true
+
+fire_order += inscribed
+pickup_thrown = false
+enable_recast_spell = true
+
+show_gold_turns = true
+show_game_turns = true
+
+# In theory it should help in autoexploring
+# EDIT: And it does. A lot!
+explore_wall_bias = 0
+explore_improved = true
+
+
+: if you.god():find("Cheibriados") then
+	hp_warning = 65
+	mp_warning = 30
+	autofight_stop = 50
+: else
+	hp_warning = 40
+	mp_warning = 30
+	autofight_stop = 40
+:end
+
+
+# Tags: hp, tab, autofight, warning,
+#####################
+##################### End of mechanics
+#####################
+
+
+#####################
+##################### Autoinscription
+#####################
+# Great thanks to bart for his/her RC and/or true author of this part.
+
+ai := autoinscribe
+
+autoinscribe_cursed = false
+
+ai += bad_item.*potion:!q
+ai += potion.*mutation:!q
+ai += wand of heal wounds:!V
+ai += wand of teleportation:!V
+ai += wand of haste:!V
+ai += wand of invisibility:!V
+ai += trishula:!d
+
+ai += staff of air:rElec
+ai += staff of cold:rC+
+ai += staff of death:rN+
+ai += staff of energy:channel
+ai += staff of fire:rF+
+ai += staff of poison:rPois
+ai += staff of power:MP+15
+ai += staff of wizardry:Wiz+
+
+ai += fire dragon (armour|hide):rF++ rC-
+ai += gold dragon (armour|hide):rC+ rF+ rPois
+ai += ice dragon (armour|hide):rC++ rF-
+ai += mottled dragon (armour|hide):rSticky
+ai += pearl dragon (armour|hide):rN+
+ai += quicksilver dragon armour:MR+
+ai += shadow dragon (armour|hide):Stlth++++
+ai += steam dragon (armour|hide):rSteam
+ai += storm dragon (armour|hide):rElec
+ai += swamp dragon (armour|hide):rPois
+ai += troll (hide|leather armour):regen
+
+ai += (leather armour|mail|plate armour|robe|skin) of cold resistance:rC+
```

Comment:
There was [^l] prefix for leather armour, but when you get that troll leather wit rC+ enchant, it won't get regen inscription.